doc/2d_combine.rst

```diff
@@ -0,0 +1,103 @@
+.. _2d_combine:
+
+=========================
+Combine Science Exposures
+=========================
+
+Overview
+========
+
+PypeIt can combine (without weighting) multiple science exposures
+as part of the data reduction process. See :ref:`coadd2d` for a weighted
+coadd, which is done outside of the data reduction pipeline.
```

Comment:
I would say here "by a separate script" or "not via run_pypeit" rather than "outside of the data reduction pipeline" since I think of PypeIt as the pipeline.

Reply:
Done!